parsl/executors/high_throughput/executor.py

```diff
@@ -268,6 +275,13 @@ def __init__(self,
         self.address = address
         self.address_probe_timeout = address_probe_timeout
         self.manager_selector = manager_selector
+        self.enable_ipv6 = enable_ipv6
+        if self.enable_ipv6:
+            # This will force usage of IPV6 for internal communication
+            self._internal_address = "[::1]"
```

Comment:
Consider renaming this attribute to `self._loopback_address`.

Reply:
I've made changes here to this effect.